internal/cmd/backup.go

```diff
@@ -118,6 +120,39 @@ func runBackup(c *cli.Context) error {
 		}
 	}
 
+	// Repositories exclude mirror repositories
+	if !c.Bool("exclude-mirror-repos") && !c.Bool("database-only") {
+		reposFile := filepath.Join(rootDir, "non-mirror-repositories.zip")
+		reposInDB, err := db.GetRepositoriesMirror()
+		if err != nil {
+			log.Fatal("Mirror repositories failed to load %v", err)
+		}
+		// Check if file path to repo e.g. foo/bar/baz.git
+		// contains repo owner and name e.g. bar/baz
+		compare := func(repos []*db.Repository, f string) bool {
+			for _, r := range repos {
+				if strings.Contains(f, fmt.Sprintf("%s/%s", r.Owner.Name, r.Name)) {
+					return true
+				}
+			}
+			return false
+		}
```

Comment:
It function will have really poor performance, imagine you have 10k repos :)

Reply:
What if iterate over strings instead of Repository records? I cant find another way...